### DISCUSSION TITLE
在jsx文件下，定义的尺寸，为什么不会自动转换为rpx

### DISCUSSION THREAD

**Discussion by zhushenhang** [NervJS/taro#13748] [Q & A] [answered]
我在scss文件下，比如定义
`padding:40px`
在微信开发者工具里可以看到自动转换为rpx，
但是在jsx文件下定义style
`padding:40px`
在工具里看到的却是40px，没有自动转换。
这个jsx文件下的样式自动转换单位，请问是需要配置吗？还是说手动写单位？

**Comment by TheKonka** [comment, marked as answer]
https://docs.taro.zone/docs/size#api

**Reply by zhushenhang** [reply]
谢谢

**Comment by Xiakeming97** [comment]
我也遇到类似的问题
<img width="391" alt="image" src="https://user-images.githubusercontent.com/111333326/236149734-c464c45f-0f50-4c94-8e95-9b95b38ffb85.png">
```
        <View style={{
          width:'375px',
          height:'20px',
          backgroundColor:'pink'
        }}></View>
```

**Reply by Xiakeming97** [reply]
```
<View style={{
      width:Taro.pxTransform(375),
       height:'20px',
       backgroundColor:'pink'
}}></View>
```
解决了
